### PR TITLE
21.7 fb compliance entry addendum

### PR DESCRIPTION
**Pull request by Ohsudev**
#### Comments:
Added missing files into this feature branch after committing default build#127  into Prime UAT generated errors.  Feature branch #255 was last feature branch committed into default #127

### CODE REVIEW

File: EHR_ComplianceDB/resources/schemas/dbscripts/sqlserver/EHR_ComplianceDB-12.40-12.41.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+ALTER TABLE ehr_compliancedb.RequirementsPerCategory add taskid ENTITYID;
+
+
+CREATE TABLE ehr_compliancedb.EmployeePerUnit
+(
+    RowId INT IDENTITY(1,1) NOT NULL,
+    EmployeeId varchar(255) not null,
+    unit varchar(255) null,
+    category varchar(255) null,
```
Comment (labkey-jeckels) [inline EHR_ComplianceDB/resources/schemas/dbscripts/sqlserver/EHR_ComplianceDB-12.40-12.41.sql]:
```suggestion
    EmployeeId nvarchar(255) not null,
    unit nvarchar(255) null,
    category nvarchar(255) null,
```

File: EHR_ComplianceDB/resources/schemas/dbscripts/sqlserver/EHR_ComplianceDB-12.40-12.41.sql [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+ALTER TABLE ehr_compliancedb.RequirementsPerCategory add trackingflag varchar(100);
```
Comment (labkey-jeckels) [inline EHR_ComplianceDB/resources/schemas/dbscripts/sqlserver/EHR_ComplianceDB-12.40-12.41.sql]:
```suggestion
ALTER TABLE ehr_compliancedb.RequirementsPerCategory add trackingflag nvarchar(100);
```

**Review by labkey-jeckels — APPROVED** [review]
We should use NVARCHAR but otherwise looks good.

**Review by kollil — COMMENTED** [review]
Looks good.

**Pull request merged**